cartography/intel/aws/organizations.py

```diff
@@ -88,12 +88,18 @@ def load_aws_accounts(neo4j_session, aws_accounts, aws_update_tag, common_job_pa
     MERGE (aa:AWSAccount{id: {ACCOUNT_ID}})
     ON CREATE SET aa.firstseen = timestamp()
     SET aa.lastupdated = {aws_update_tag}, aa.name = {ACCOUNT_NAME}
+    WITH aa
+    MERGE (aa)-[:RESOURCE]->(root:AWSPrincipal{arn: {RootArn})
```

Comment:
I think we need to set the `lastupdated` and `firstseen` on the `:RESOURCE` relationship here.